web/src/stream_popover.ts

```diff
@@ -42,6 +43,8 @@ import * as util from "./util.ts";
 // that pop up from the left sidebar.
 let stream_widget_value: number | undefined;
 let move_topic_to_stream_topic_typeahead: Typeahead<string> | undefined;
+const ls_key_propagate_mode_prefix = "last_propagate_mode_for_conversation_";
+const ls = localstorage();
```

Comment:
I think this should just be a JS variable, like `stream_widget_value`. It doesn't seem relevant to save across browser sessions, and polluting browser local storage has costs, including coming up with a good name; I don't like `last_propagate_mode_for_conversation_`.